.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+[//]: # (Thanks for opening this pull request! Before you proceed please make sure that you have an issue that explains what this pull request will do.
+         Make sure that all your commits link to this issue e.g. "My commit \(bunq/sdk_java#<issue nr>\)".
```

Comment:
Either make point in regex optional or make sure your example here matches it. 😄

Reply:
Good one 😂